frontend/src/components/Start.js

```diff
@@ -0,0 +1,229 @@
+import styled from 'styled-components/macro'
+import startscreenJpg from './../assets/startscreen.jpg'
+import wandergoldSvg from './../assets/wandergold.svg'
+import closeSvg from './../assets/close.svg'
+import compassSvg from './../assets/compass.svg'
```

Comment:
It is great that you have a naming convention here 👍.
My only issue with this is, that the values of these variables are strings, which might be a little bit misleading. 

Wdyt about this?

```suggestion
import startscreenSrc from './../assets/startscreen.jpg'
import wandergoldSrc from './../assets/wandergold.svg'
import closeSrc from './../assets/close.svg'
import compassSrc from './../assets/compass.svg'
```

Reply:
I would never have thought of "Src", but why not?